docs/management_api.yml

```diff
@@ -513,6 +513,12 @@ paths:
           format: integer
           default: 20
           maximum: 500
+        - name: sort_by_artifact_size
```

Comment:
Q: why a flag (boolean) instead of a `sort` parameter with `artifact_size` as value? I'm asking because the `sort` parameter is more future proof than the flag.